### PR TITLE
Google Analytics Snippet on header.php

### PR DESCRIPTION
**Pull request by eabquina**
To cover #70

### CODE REVIEW

File: web/wp-content/themes/dctx/header.php [hunk 1]
```diff
@@ -11,6 +11,16 @@
 	<link rel="icon" type="image/png" href="<?php echo ROOT; ?>favicon.png">
 
 	<?php wp_head(); ?>
+
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162125973-1"></script>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/header.php:16]: :no_entry_sign: **Error**: Scripts must be registered/enqueued via wp_enqueue_script (*WordPress.WP.EnqueuedResources.NonEnqueuedScript*).

File: web/wp-content/themes/dctx/header.php [hunk 2]
```diff
@@ -11,6 +11,16 @@
 	<link rel="icon" type="image/png" href="<?php echo ROOT; ?>favicon.png">
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/header.php:11]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'ROOT' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Comment by carl-alberto** [comment]
@eabquina this might be better https://wordpress.org/plugins/host-analyticsjs-local/?

File: web/wp-content/themes/dctx/header.php [hunk 3]
```diff
@@ -11,6 +11,7 @@
...
 
 	<?php wp_head(); ?>
+
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/header.php:14]: :warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/functions.php [hunk 4]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
+        true // if true, the script is placed before the  end tag
+    );
+    // Enqueue the registered script file
+    wp_enqueue_script('google-analytics');
+}
+add_action('wp_enqueue_scripts', 'add_google_analytics');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:277]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:277]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: web/wp-content/themes/dctx/functions.php [hunk 5]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
+        true // if true, the script is placed before the  end tag
+    );
+    // Enqueue the registered script file
+    wp_enqueue_script('google-analytics');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:275]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:275]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:275]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 6]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
+        true // if true, the script is placed before the  end tag
+    );
+    // Enqueue the registered script file
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:274]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:274]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 7]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
+        true // if true, the script is placed before the  end tag
+    );
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:273]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 8]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
+        true // if true, the script is placed before the  end tag
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:272]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:272]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 9]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
+        '1.0', // version number
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:271]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:271]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 10]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
...
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
+        false, // no dependencies
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:270]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:270]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 11]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
+        get_stylesheet_directory_uri() . '/analyticstracking.js', // location of your file
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:269]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:269]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 12]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
+        'google-analytics', // handle name referred to in the "wp_enqueue_script" call below
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:268]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:268]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 13]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
+    wp_register_script(
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:267]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 14]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
+    // Register analyticstracking.js file (Google Analytics)
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:266]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:266]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: web/wp-content/themes/dctx/functions.php [hunk 15]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
+function add_google_analytics() {
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:265]: :no_entry_sign: **Error**: You must use &quot;/**&quot; style comments for a function comment (*Squiz.Commenting.FunctionComment.WrongStyle*).

File: web/wp-content/themes/dctx/functions.php [hunk 16]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
 add_editor_style( 'style-editor.css' );
 
+
+// Function to add analyticstracking.js to the site
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:264]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

File: web/wp-content/themes/dctx/functions.php [hunk 17]
```diff
@@ -260,4 +260,20 @@ function additional_securityheaders( $headers ) {
 add_theme_support('editor-styles');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:260]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/functions.php:260]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 18]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-162125973-1', 'auto');
+  ga('send', 'pageview');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:7]: :no_entry_sign: **Error**: File must end with a newline character (*Generic.Files.EndFileNewline.NotFound*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:7]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:7]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:7]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 19]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-162125973-1', 'auto');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:6]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:6]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:6]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 20]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:4]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:4]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:4]: :no_entry_sign: **Error**: Line indented incorrectly; expected 0 tabs, found 0 (*Generic.WhiteSpace.ScopeIndent.IncorrectExact*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:4]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 21]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :no_entry_sign: **Error**: Expected 1 space after &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :no_entry_sign: **Error**: Expected 1 space before &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :warning: **Warning**: Equals sign not aligned correctly; expected 1 space but found 0 spaces (*Generic.Formatting.MultipleStatementAlignment.IncorrectWarning*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :no_entry_sign: **Error**: Line indented incorrectly; expected at least 1 tabs, found 0 (*Generic.WhiteSpace.ScopeIndent.Incorrect*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:3]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 22]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space after &quot;*&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space before &quot;*&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 0 spaces (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space after &quot;||&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space before &quot;||&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space after &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Expected 1 space before &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :warning: **Warning**: Equals sign not aligned correctly; expected 1 space but found 0 spaces (*Generic.Formatting.MultipleStatementAlignment.IncorrectWarning*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :no_entry_sign: **Error**: Line indented incorrectly; expected at least 1 tabs, found 0 (*Generic.WhiteSpace.ScopeIndent.Incorrect*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:2]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/analyticstracking.js [hunk 23]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :no_entry_sign: **Error**: Expected 1 space after &quot;||&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :no_entry_sign: **Error**: Expected 1 space before &quot;||&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :no_entry_sign: **Error**: Expected 1 space after &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :no_entry_sign: **Error**: Expected 1 space before &quot;=&quot;; 0 found (*WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 1 space but found 0 spaces (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/analyticstracking.js:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

File: web/wp-content/themes/dctx/header.php (hunk 2 above)
Comment (eabquina) [reply web/wp-content/themes/dctx/header.php:11]: skip as done from prev

**Comment by eabquina** [comment]
@carl-alberto closing na this, lets use the plugin na

**Pull request closed without merging**